.changeset/plenty-humans-turn.md

```
@@ -0,0 +1,5 @@
+---
+"data-dashboard": minor
+---
+
+Add unique index for combination wallet type, wallet address and public key (non-EVM wallets are using them to check ownership)
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)